radio/src/datastructs.h

```diff
@@ -679,8 +679,7 @@ PACK(struct ModuleData {
       uint8_t receiver_telem_off:1;     // false = receiver telem enabled
       uint8_t receiver_channel_9_16:1;  // false = pwm out 1-8, true 9-16
       uint8_t external_antenna:1;       // false = internal antenna, true = external antenna
-      uint8_t sport_out:1;
-      uint8_t spare2;
+      uint8_t spare3;
```

Comment:
This does not look right. We should have spare1:1 instead sport_out. We always document spare bits

Reply:
This looks better thanks!. NO idea why github thinks my old comment is still valid